modules/python-modules/syslogng/dest.py

```diff
@@ -103,4 +103,4 @@ def send(self, msg):
         Returns:
             int: one value from the LogDestinationResult enum
         """
-        return self.SUCCESS
+        raise NotImplementedError
```

Comment:
[optional]

While we are here, we could descend from `ABC`, to show these error in init time:
```diff
--- a/modules/python-modules/syslogng/dest.py
+++ b/modules/python-modules/syslogng/dest.py
@@ -20,6 +20,7 @@
 # COPYING for details.
 #
 #############################################################################
+from abc import ABC, abstractmethod
 try:
     from _syslogng import LogDestination, LogDestinationResult

@@ -39,7 +40,7 @@ except ImportError:
         NOT_CONNECTED = auto(),


-class LogDestination(LogDestination):
+class LogDestination(ABC, LogDestination):
     DROP = LogDestinationResult.DROP
     ERROR = LogDestinationResult.ERROR
     SUCCESS = LogDestinationResult.SUCCESS
@@ -91,6 +92,7 @@ class LogDestination(LogDestination):
         """
         pass

+    @abstractmethod
     def send(self, msg):
         """Send a message to the target service
```

Reply:
I would rather avoid multiple inheritance. I had a quick look at metaclasses too...